docs/topics/proc_deploying-the-configmap-booster-using-the-oc-cli-client.adoc

```diff
@@ -132,7 +132,7 @@ ifdef::configmap-nodejs[]
 $ npm install && npm run openshift
 ----
 +
-These commands install any missing package dependencies, then using the xref:about-nodeshift[Nodeshift] package, launch the link:{link-s2i-process}[S2I process] on OpenShift to start the pod.
+These commands install any missing package dependencies, then using the xref:about-nodeshift[Nodeshift] package, deploy the booster on OpenShift.
```

Comment:
can we change the word package to module

the other boosters will need this update to.  just tagging this one

Reply:
nice catch, sorry i missed this.

Thank you for reviewing.